nva-datamodel-java/src/main/java/no/unit/nva/model/Publication.java

```diff
@@ -317,7 +319,11 @@ private void verifyStatusTransition(PublicationStatus nextStatus)
             throw new InvalidPublicationStatusTransitionException(currentStatus, nextStatus);
         }
     }
-    
+
+    public List<File> getAssociatedArtifacts() {
+        return new ArrayList<>(getFileSet().getFiles());
```

Comment:
if getFileSet returns null you will get a null pointer exception

Reply:
Fixed, good catch